### PR TITLE
Improved static code analysis

### PR DESCRIPTION
**Pull request by chives**

### CODE REVIEW

**Comment by chives** [comment]
@fsi-open

File: Admin/CRUD/Context/ListElementContext.php [hunk 1]
```diff
@@ -37,6 +37,9 @@ class ListElementContext extends ContextAbstract
      */
     protected $dataGrid;
 
+    /**
+     * @param ListElement $element
+     */
     public function setElement(Element $element): void
     {
         $this->element = $element;
```
Comment (szymach) [inline Admin/CRUD/Context/ListElementContext.php:45]: No assertion here?

File: Admin/CRUD/GenericCRUDElement.php [hunk 2]
```diff
@@ -74,7 +74,11 @@ public function configureOptions(OptionsResolver $resolver): void
         $resolver->setNormalizer('template_crud_create', function (Options $options, $value) {
             if ($value !== $options['template_crud_edit']) {
                 throw new RuntimeException(
-                    'CRUD admin element options "template_crud_create" and "template_crud_edit" have both to have the same value'
+                    sprintf(
```
Comment (szymach) [inline Admin/CRUD/GenericCRUDElement.php:77]: Why make this a formatted string? These are scalar values.

File: Admin/CRUD/Context/ListElementContext.php (hunk 1 above)
Comment (chives) [reply Admin/CRUD/Context/ListElementContext.php:45]: typehint in docblock is enough for phpstan

File: Admin/CRUD/GenericCRUDElement.php (hunk 2 above)
Comment (chives) [reply Admin/CRUD/GenericCRUDElement.php:77]: to make shorter lines :]

File: Admin/CRUD/Context/ListElementContext.php (hunk 1 above)
Comment (szymach) [reply Admin/CRUD/Context/ListElementContext.php:45]: Well we should aim for consistency.

File: Admin/CRUD/GenericCRUDElement.php (hunk 2 above)
Comment (szymach) [reply Admin/CRUD/GenericCRUDElement.php:77]: Can't you just split the line in two?

**Review by szymach — APPROVED** [review]

**Pull request merged**